packages/react-google-maps-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@react-google-maps/api",
-  "version": "1.0.8",
+  "version": "1.1.0",
```

Comment:
You do not need to update package version. Now I do it with `lerna publish` in root, and it bumps version automagically

Reply:
I see. And do you tell it to bump a major/minor or patch? Because I think this should be a minor version